### PR TITLE
add RNAsum button

### PR DESCRIPTION
**Pull request by williamputraintan**
- Added RNAsum trigger button via portal api `/manops/rnasum` route
- Import list of RNAsum datasets from  [GitHub](https://github.com/umccr/RNAsum/blob/master/TCGA_projects_summary.md#pan-cancer-dataset)

### CODE REVIEW

**Review by victorskl — APPROVED** [review]
Nice! Kindly follow up testing in DEV, please. Thank you.

File: src/utils/rnasum.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    tissue_code: '1',
+    samples_no: '151',
+  },
+];
+
+export const PAN_CANCER_DATASETS_OPTION = [
+  {
+    no: '1',
+    project: 'PANCAN',
+    name: 'Samples from all [33 cancer types](#primary-datasets), 10 samples from each',
```
Comment (pdiakumis) [inline src/utils/rnasum.js:248]: I'd probably get rid of the markdown link here ;-)

**Review by victorskl — COMMENTED** [review]
Hey Will; as discussed, just added UI validation requirement [in ticket](https://github.com/umccr/data-portal-client/issues/137#issuecomment-1203326467). Sorry. I should have added that early.

You can continue with this PR. Or, feel free to make another PR for that. I will leave it to you!

**Comment by williamputraintan** [comment]
Yup, all good! Will update it here and merge it as one PR.

File: src/utils/rnasum.js (hunk 1 above)
Comment (williamputraintan) [reply src/utils/rnasum.js]: Whoops! Thanks for checking this out 😅

File: src/components/LaunchRNAsumReport.tsx [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+      setTriggerStatus((prevState) => {
+        return {
+          ...prevState,
+          isLoading: false,
+          isError: true,
+        };
+      });
+    }
+  };
+  console.log(rnasumInputData);
```
Comment (victorskl) [inline src/components/LaunchRNAsumReport.tsx:174]: May be remove stray console log?

File: src/components/LaunchRNAsumReport.tsx [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+                      <TableRow>
+                        <TableCell>
+                          <Typography color='error'>ERROR</Typography>
+                        </TableCell>
+                        <TableCell>
+                          <Typography color='error' align='right'>
+                            {rnasumInputData.wtsBamsIca.length < 1
+                              ? `No WTS bam file found in GDS`
+                              : rnasumInputData.wgsCancer.length < 1
+                              ? `No WGS bam file found in GDS`
```
Comment (victorskl) [inline src/components/LaunchRNAsumReport.tsx:240]: Can you change it to say `No umccrise workflow output found for the Subject` instead, pls.

File: src/components/LaunchRNAsumReport.tsx [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+                    {rnasumInputData.wtsBamsIca.length < 1 ||
+                    rnasumInputData.wgsCancer.length < 1 ? (
+                      <TableRow>
+                        <TableCell>
+                          <Typography color='error'>ERROR</Typography>
+                        </TableCell>
+                        <TableCell>
+                          <Typography color='error' align='right'>
+                            {rnasumInputData.wtsBamsIca.length < 1
+                              ? `No WTS bam file found in GDS`
```
Comment (victorskl) [inline src/components/LaunchRNAsumReport.tsx:238]: Also. `No transcriptome workflow output found for the Subject` instead, pls.

File: src/components/LaunchRNAsumReport.tsx (hunk 2 above)
Comment (williamputraintan) [reply src/components/LaunchRNAsumReport.tsx:174]: 🤦‍♂️ got that slipped away

**Pull request merged**